modules/nf-core/porechop/abi/main.nf

```diff
@@ -0,0 +1,39 @@
+process PORECHOP_ABI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::porechop_abi=0.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/porechop_abi:0.5.0--py310h590eda1_0':
+        'quay.io/biocontainers/porechop_abi:0.5.0--py310h590eda1_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val abi
```

Comment:
This can be supplied using `ext.args`. See [here](https://nf-co.re/docs/contributing/modules#general) paragraph 1